app/react/components/BoxSelector/BoxSelectorItem.css

```diff
@@ -13,8 +13,7 @@
   color: var(--text-boxselector-header);
 }
 
-.boxselector_header .fa,
-.fab {
+.boxselector_header {
```

Comment:
this css rule is related to icons. should be removed totally

Reply:
updated